en/docs/try-out/account-information-service-flow.md

```diff
@@ -0,0 +1,896 @@
+
+5. Click **Next**.
+
+6. Set the context to `/xs2a` and leave the endpoint empty. ![set-context](../assets/img/try-out/account-flow/set-context.png)
+
+7. Click **Create** to create the API. ![create-accounts](../assets/img/get-started/quick-start-guide/create-accounts.png)
+
+8. After the API is successfully created, go to **Portal Configurations** using the left menu panel. ![portal-configurations](../assets/img/get-started/quick-start-guide/portal-configurations.png)
+
+9. Select **Subscriptions** from the left menu pane and set the business plan to **Unlimited: Allows unlimited requests**. ![business-plan](../assets/img/get-started/quick-start-guide/business-plan.png)
+
+10. Click **Save**.
+
+11. Select **Runtime** from the left menu pane.
+
+12. Toggle the **Schema Validation** button to enable Schema Validation. ![schema-validation](../assets/img/get-started/quick-start-guide/schema-validation.png)
+
+13. Click the **Edit** button under **Request > Message Mediation**. ![edit_message_mediation](../assets/img/get-started/quick-start-guide/edit-message-mediation.png)
+
+14. Select the **Custom Policy** option.
+
+15. Upload the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources/apis/berlin.group.org/PSD2API_1.3.6/dynamic-endpoint-insequence-1.3.6.xml` insequence file. ![uplaoad_insequence](../assets/img/try-out/account-flow/upload-insequence.png)
+ 
+16. Click **Select**. 
+
+17. Scroll down and click **SAVE**.
+
+18. Use the left menu panel and go to **API Configurations > Endpoints**. 
+
+    ![select_endpoints](../assets/img/get-started/quick-start-guide/select-endpoints.png)
+
+19. Add a **Dynamic Endpoint**. ![add_dynamic_endpoint](../assets/img/get-started/quick-start-guide/add_dynamic_endpoint.png)
+
+20. Go to **Deployments** using the left menu pane. 
+
+    ![select_deployments](../assets/img/get-started/quick-start-guide/select-deployments.png)
+    
+21. Select the API Gateway type, in this scenario, it is **Default**. ![api_gateway](../assets/img/get-started/quick-start-guide/dcr-api-gateway.png)
+
+22. Click **Deploy**.
+
+23. Go to **Overview** using the left menu pane. 
+
+    ![select_overview](../assets/img/get-started/quick-start-guide/select-overview.png)
+
+24. Click **Publish**. ![publish_api](../assets/img/get-started/quick-start-guide/publish-api.png)
+
+## Subscribing to PSD2 API
+
+1. The deployed API is now available in the Developer Portal at `https://<APIM_HOST>:9443/devportal`.
+
+2. Select the **NextGenPsd2XS2AFramework v1** API.
+ 
+3. Locate **Subscriptions** from the left menu pane. 
+
+    ![select_subscriptions](../assets/img/get-started/quick-start-guide/select-subscriptions.png)
+    
+4. From the **Application** dropdown, select the application that you want to be subscribed to the NextGenPsd2XS2AFramework v1 API. ![subscribe_to_api](../assets/img/try-out/account-flow/subscribe-to-api.png)
+
+5. Click **Subscribe**.
+
+## Invoking PSD2 API
```

Comment:
Is it OK to call this the PSD2 API here since then it would be the same for payments flow as well?

Reply:
will copy the steps to deploy to a common place

Reply:
Fixed in 49adb6d8c05809680c301f250e674df1f369b3f0 & 2d265ec7996df3d487c08ddc78bbc2d6bac52487